src/ui/reducers/network.ts

```diff
@@ -22,6 +22,7 @@ export type NetworkState = {
   requestBodies: Record<string, RequestBodyData[]>;
   requests: RequestInfo[];
   selectedRequestId: string | null;
+  isCopyTriggered: boolean;
```

Comment:
Doesn't look like anything is reading or updating this piece of reducer state?

Reply:
Same reason as above. Missed removing the old changes